ci/run-tests.sh

```diff
@@ -16,6 +16,7 @@ exec docker run -it --rm \
 	-e TOX=true \
 	-e MU_APPLICATION_GRAPH=http://mu.semte.ch/test \
 	-e MU_SPARQL_ENDPOINT=http://delta:8890/sparql \
+	-e POLL_RETRIES=15 \
```

Comment:
Check with the tests if it doesn't waste too much time for no real purpose because all the services are executed on the same machine, we don't expect the database to be unreachable for example. I suppose a value of 1 would be enough.